stdlib/abc.pyi

```diff
@@ -28,17 +28,17 @@ class ABCMeta(type):
     def register(cls: ABCMeta, subclass: type[_T]) -> type[_T]: ...
 
 def abstractmethod(funcobj: _FuncT) -> _FuncT: ...
-@deprecated("Use 'classmethod' with 'abstractmethod' instead")
+@deprecated("Deprecated since Python 3.3. Use `classmethod` with `abstractmethod` instead.")
```

Comment:
You have to put the classmethod decorator on top of the abstractmethod decorator in order for this to work, but I can never remember which way round it has to go when I'm actually writing Python code. Maybe we could put that in the deprecation message?

```suggestion
@deprecated("Deprecated since Python 3.3. Use `classmethod` stacked on top of `abstractmethod` instead.")
```